crates/services/sync/src/import.rs

```diff
@@ -270,59 +338,13 @@ async fn wait_for_notify_or_shutdown(
     matches!(r, futures::future::Either::Left(_))
 }
 
-/// Returns a stream of headers processing concurrently up to `max_get_header_requests`.
-/// The headers are returned in order.
-fn get_header_range_buffered(
-    range: RangeInclusive<u32>,
-    params: &Config,
-    p2p: Arc<impl PeerToPeerPort + Send + Sync + 'static>,
-) -> impl Stream<Item = anyhow::Result<SourcePeer<SealedBlockHeader>>> {
-    get_header_range(range, p2p)
-        .buffered(params.max_get_header_requests)
-        // Continue the stream unless an error or none occurs.
-        .into_scan_none_or_err()
-        .scan_none_or_err()
-}
-
-#[tracing::instrument(skip(p2p))]
-/// Returns a stream of network requests for headers.
-fn get_header_range(
-    range: RangeInclusive<u32>,
+async fn get_header_range_single_req(
+    range: &RangeInclusive<u32>,
     p2p: Arc<impl PeerToPeerPort + 'static>,
-) -> impl Stream<
-    Item = impl Future<Output = anyhow::Result<Option<SourcePeer<SealedBlockHeader>>>>,
-> {
-    stream::iter(range).map(move |height| {
```

Comment:
You can use the old naming instead of creating new functions with slightly different names =) It will make fewer conflicts for @bvrooman and still works for you. Instead of `get_header_range`, you can use `get_header_range_buffered`, and instead of `get_header_range_single_req`, you can use `get_header_range`.


Also, it seems you don't need to touch other code except `get_header_range`. In this case, your change will be super tiny